Tools/LyTestTools/tests/unit/test_editor_test_utils.py

```diff
@@ -119,15 +119,15 @@ def test_RetrieveEditorLogContent_CrashLogExists_ReturnsLogInfo(self, mock_retri
         mock_log = 'mock log info'
 
         with mock.patch('builtins.open', mock.mock_open(read_data=mock_log)) as mock_file:
-            assert f'[editor.log]  {mock_log}' == editor_test_utils.retrieve_editor_log_content(0, mock_logname, mock_workspace)
+            assert f'[{mock_logname}]  {mock_log}' == editor_test_utils.retrieve_editor_log_content(0, mock_logname, mock_workspace)
```

Comment:
I would prefer if this assert would be:
`assert mock_log in editor_test_utils.retrieve_editor_log_content(0, mock_logname, mock_workspace)`